packages/pocl-job/src/staging/create-transactions.js

```diff
@@ -71,7 +71,7 @@ const getInitialState = async filename => {
 const createTransactionsInSalesApi = async (filename, state) => {
   if (state.buffer.length) {
     const createResults = await salesApi.createTransactions(state.buffer.map(item => item.createTransactionPayload))
-
+    
```

Comment:
This extra space can be removed

Reply:
I ran the `prettier-standard` script when I noticed this, but looking at it now, I didn't push my changes up! This has now been removed :)